README.md

```diff
@@ -1,6 +1,7 @@
 <img src="https://github.com/opensearch-project/documentation-website/blob/main/assets/images/logo.svg" height="64px"/>
 
-Hello!
+We believe that open source makes the world a better place. Sadly, [Elastic
```

Comment:
I feel conflicted about these lines and would appreciate your feedback. On the one hand, I understand this to be what project is about. The open source initiative is very clear in their statement, and it makes for an evocative opening. On the other hand, it can be interpreted as enmity and I don't think this is the place to perpetuate the disagreements between Elastic and AWS.

What do you think about adding a **Controversy** section at the end of the README? While we don't want the repository to be a place for political discussions, it's important to understand (and acknowledge) the surrounding context and history.

Reply:
I would keep the controversy out, reserve judgement, and state facts only: licensing changes and project renaming.

Reply:
I agree with @dblock. No need to introduce controversy.